src/main/paradox/docs/api/iam/iam-acls-api.md

```diff
@@ -1 +1,206 @@
+
+This operation creates a collection of ACL on the provided path.
+```
+PUT /v1/acls/{address}
+  {...}
+```
+...where `{address}` is the target path for the ACL collection.
+
+The json payload contains the collection of ACL to set.
+
+**Example**
+
+Request
+:   @@snip [acls-add.sh](../assets/acls-add.sh)
+
+Payload
+:   @@snip [acls-add.json](../assets/acls-add.json)
+
+Response
+:   @@snip [acls-added-ref.json](../assets/acls-added-ref.json)
+
+
+## Replace ACLs
+
+This operation overrides the collection of ACL on the provided path.
+```
+PUT /v1/acls/{address}?rev={previous_rev}
+  {...}
+```
+
+...where:
+
+- `{previous_rev}`: Number - the last known revision for the ACL collection.
+- `{address}`: String - is the target path for the ACL collection.
+
+The json payload contains the collection of ACL to set.
+
+**Example**
+
+Request
+:   @@snip [acls-replace.sh](../assets/acls-replace.sh)
+
+Payload
+:   @@snip [acls-add.json](../assets/acls-add.json)
+
+Response
+:   @@snip [acls-replaced-ref.json](../assets/acls-replaced-ref.json)
+
+
+## Subtract ACLs
+
+This operation removes the provided ACL collection from the existing collection of ACL on the provided path.
+
+```
+PATCH /v1/acls/{address}?rev={previous_rev}
+  {...}
+```
+...where:
+
+- `{previous_rev}`: Number - the last known revision for the ACL collection.
+- `{address}`: String - is the target path for the ACL collection.
+ 
+The json payload contains the collection of ACL to remove.
+
+**Example**
+
+Request
+:   @@snip [acls-subtract.sh](../assets/acls-subtract.sh)
+
+Payload
+:   @@snip [acls-subtract.json](../assets/acls-subtract.json)
+
+Response
+:   @@snip [acls-subtracted-ref.json](../assets/acls-subtracted-ref.json)
+
+## Append ACLs
+
+This operation appends the provided ACL collection to the existing collection of ACL on the provided path.
+
+```
+PATCH /v1/acls/{address}?rev={previous_rev}
+  {...}
+```
+...where:
+
+- `{previous_rev}`: Number - the last known revision for the ACL collection.
+- `{address}`: String - is the target path for the ACL collection.
+
+The json payload contains the collection of ACL to add.
+
+**Example**
+
+Request
+:   @@snip [acls-append.sh](../assets/acls-append.sh)
+
+Payload
+:   @@snip [acls-append.json](../assets/acls-append.json)
+
+Response
+:   @@snip [acls-appended-ref.json](../assets/acls-appended-ref.json)
+
+
+## Delete ACLs
+
+This operation deletes the entire collection of ACL on the provided path.
+
+```
+DELETE /v1/acls/{address}?rev={previous_rev}
+```
+
+...where:
+ 
+- `{previous_rev}`: Number - the last known revision for the ACL collection.
+- `{address}`: String - is the target path for the ACL collection.
+
+Request
+:   @@snip [acls-delete.sh](../assets/acls-delete.sh)
+
+Response
+:   @@snip [acls-deleted-ref.json](../assets/acls-deleted-ref.json)
+
+
+## Fetch ACLs
+
+```
+GET /v1/acls/{address}?rev={rev}&self={self}
+```
+
+...where 
+
+- `{address}`: String - is the target path for the ACL collection.
```

Comment:
we need to decide on whether we want to use `address` or `path`, using both terms is confusing.
@MFSY, how should we refer to the "address" where we set ACLs (e.g. /, /org, /org/proj)?
I would recommend `address` because it signifies a resource address.